stable/xray/values.yaml

```diff
@@ -274,10 +282,44 @@ rabbitmq:
     # existingErlangSecret: <name-of-existing-secret>
   service:
     port: 5672
+  external:
+    rabbitmqUrl:
+    rabbitmqUsername:
+    rabbitmqPassword:
+    rabbitmqErlangCookie:
   persistence:
     enabled: true
     accessMode: ReadWriteOnce
     size: 20Gi
+  ## Load Definitions - https://www.rabbitmq.com/management.html#load-definitions
+  # ref : https://github.com/bitnami/charts/tree/master/bitnami/rabbitmq#load-definitions
+  extraSecrets:
+    load-definition:
+      load_definition.json: |
+        {
+          "vhosts": [
+            {
+              "name": "/"
+            }
+          ],
+          "policies": [
+            {
+              "name": "ha-all",
+              "apply-to": "all",
+              "pattern": ".*",
+              "vhost": "/",
+              "definition": {
+                "ha-mode": "all",
+                "ha-sync-mode": "automatic",
+              }
+            }
+          ]
+        }
+  loadDefinition:
+    enabled: true
+    existingSecret: load-definition
```

Comment:
How is this expected to exist? Where is this secret supposed to be created?

Reply:
More Info - https://github.com/bitnami/charts/tree/master/bitnami/rabbitmq#load-definitions
https://github.com/bitnami/charts/blob/master/bitnami/rabbitmq/templates/secrets.yaml#L25

Reply:
Your link doesn’t explain where the secret comes from... they say you can use a pre-created secret or provide the configuration to the chart in the values and let it create the secret for you. I don’t see how you expect this secret to exist if we don’t create it before. The name indicates this is external to the chart and if you want to use it, it needs to be created beforehand.

Reply:
Still, a user is very likely to not provide the external rabbit configuration here and get very poor UX

Reply:
I was referring  to 2nd section  https://github.com/bitnami/charts/tree/master/bitnami/rabbitmq#load-definitions

If needed, you can use extraSecrets to let the chart create the secret for you. This way, you don't need to manually create it before deploying a release. For example :
`extraSecrets:
  load-definition:
    load_definition.json: |
      {
        "vhosts": [
          {
            "name": "/"
          }
        ]
      }
loadDefinition:
  enabled: true
  existingSecret: load-definition
extraConfiguration: |
  management.load_definitions = /app/load_definition.json`

Reply:
Apologies, I missed that. Has this been tested? Does it work?

Reply:
Testing - Work in Progress  :)
cc @rahulsadanandan

Reply:
👍🏻